cmd/ingest/ingest.go

```diff
@@ -27,14 +27,14 @@ type Message struct {
 func main() {
 	config := NewConfig()
 	mq := broker.New(config.Broker)
-	db, err := postgres.NewDB(config.Postgres)
+	dbs, err := postgres.NewDB(config.Postgres)
```

Comment:
Unnecessary renaming
```suggestion
	db, err := postgres.NewDB(config.Postgres)
```